drivers/pwm/pwm_stm32.c

```diff
@@ -45,10 +45,10 @@ enum capture_state {
 	CAPTURE_STATE_WAIT_FOR_PERIOD_END = 3
 };
 
-/** Return the complimentary channel number
+/** Return the complementary channel number
  * that is used to capture the end of the pulse.
  */
-static const uint32_t complimentary_channel[] = {0, 2, 1, 4, 3};
+static const uint32_t complementary_channel[] = {2, 1, 4, 3};
```

Comment:
Shouldn't this account for pre-decrement now?
```suggestion
static const uint32_t complementary_channel[] = {1, 0, 3, 2};
```

Reply:
Done